src/form/editors/impl/datalist/views/ButtonView.ts

```diff
@@ -40,10 +51,24 @@ export default TextEditorView.extend({
         }
     },
 
-    events: false,
+    events() {
+        const events = {
```

Comment:
if (!this.options.isAutocompleteMode} return {}

Reply:
done